tests/unit/hosts/test_urls.py

```diff
@@ -57,18 +63,18 @@ def test_host_urls(self):
         )
 
         url = reverse_lazy(
-            "hosts:get_challenge_host_details",
+            "hosts:get_challenge_host_delete",
             kwargs={
                 "challenge_host_team_pk": self.challenge_host_team.pk,
-                "pk": self.challenge_host.pk,
+                "challenge_host_pk": self.invite_host_user.pk,
```

Comment:
Do we have to change from `self.challenge_host.pk` to `self.invite_host_user.pk`?

Reply:
> Do we have to change from `self.challenge_host.pk` to `self.invite_host_user.pk`?

No, it works for self.challenge_host.pk as well. Thanks for pointing this out.